config.dsc

```diff
@@ -169,7 +169,6 @@ config({
                 { id: "Microsoft.Data.Edm", version: "5.8.2" },
 
                 // xUnit
-                { id: "xunit", version: "2.4.1" },

```

Comment:
is this intentional?

Reply:
It's intentional, as this is just a facade package that has no content and is not referenced by the transitive closure at all.